CMakeLists.txt

```diff
@@ -2,13 +2,21 @@ project(ffxiv-wireshark-dissector C)
 
 cmake_minimum_required(VERSION 3.7)
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
-set(CMAKE_INSTALL_LIBDIR ~/.wireshark)
+IF (WIN32)
+  set(CMAKE_INSTALL_LIBDIR $ENV{APPDATA}/Wireshark/plugins)
```

Comment:
Can this be accomplished by just setting the env on Windows? Since the plugin gets compiled into Wireshark itself and uses the Wireshark CMakeLists, I don't think the plugin CMakeLists needs to be actually modified for this.